src/services/moderators.service.js

```diff
@@ -9,6 +9,15 @@ const moderatorService = {
         catch(error){
             throw new Error(error);
         }
+    },
+    async getCurrentModerator(){
+        try{
+            const response = await RequestService.get('/api/currentModerator');
+            return response;
+        }
+        catch(error){
+            throw new Error(error);
+        }
     }
 };
 
```

Comment:
![image](https://user-images.githubusercontent.com/48828135/69957811-230bfb00-150c-11ea-8ec3-8a99dfb08eea.png)
It should work in thiis way?

Reply:
Yes we haven't avatar on this account in Postgres Database

Reply:
I add avatar to this moderator in BD